odoo/modules/registry.py

```diff
@@ -110,6 +110,7 @@ def init(self, db_name):
         self._assertion_report = assertion_report.assertion_report()
         self._fields_by_model = None
         self._post_init_queue = deque()
+        self.__cache = LRU(8192)
```

Comment:
One should also replace all occurrences of
```python
lazy_property.reset_all(self)
```
by
```python
self.__cache.clear()
```
and remove `lazy_property` from imports.

Reply:
There doesn't seem to be any reset_all on registries, only on envs and models, so this should not be relevant (logically the cache would get cleared but never nuked entirely and re-created from scratch).